sound/soc/intel/boards/sof_sdw_max98373.c

```diff
@@ -116,7 +114,6 @@ int sof_sdw_mx8373_late_probe(struct snd_soc_card *card)
 	struct snd_soc_dapm_context *dapm = &card->dapm;
 
 	/* Disable Left and Right Spk pin after boot */
-	snd_soc_dapm_disable_pin(dapm, "Left Spk");
-	snd_soc_dapm_disable_pin(dapm, "Right Spk");
+	snd_soc_dapm_disable_pin(dapm, "Spk");
```

Comment:
update the comment since there is no longer a left and right speaker pin.

Reply:
> update the comment since there is no longer a left and right speaker pin.

Done